modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -11,7 +11,7 @@
 The address pool custom resource definition (CRD) and API documented in "Load balancing with MetalLB" in {product-title} 4.10 can still be used in {product-version}. However, the enhanced functionality associated with advertising the `IPAddressPools` with layer 2 or the BGP protocol is not supported when using the address pool CRD.   
 ====
 
-The fields for the `IPAddressPool` custom resource are described in the following table.
+The fields for the `IPAddressPool` custom resource are described in the following tables.
```

Comment:
Should it be 'is' versus 'are' as it is custom resource.

Reply:
"are" refers to the "fields" so it's correct, thanks.